{{cookiecutter.project_name}}/Pipeline-Instructions.md

```diff
@@ -24,11 +24,17 @@ aws ssm put-parameter \
     --type "String" \
     --value "GITHUB_REPO_NAME"
 
-aws ssm put-parameter \
-    --name "/service/{{cookiecutter.project_name.lower().replace(' ', '-')}}/github/token" \
+aws secretsmanager create-secret --name GithubToken \
     --description "Github Token for Cloudformation Stack {{cookiecutter.project_name.lower().replace(' ', '-')}}-pipeline" \
+    --secret-string "TOKEN"
+
+ARN=$(aws secretsmanager list-secrets --query 'SecretList[?Name==`GithubToken`].{arn:ARN}' --output text)
+
+aws ssm put-parameter \
+    --name "/service/{{cookiecutter.project_name.lower().replace(' ', '-')}}/github/token_arn_suffix" \
+    --description "Secrets Manager ARN Suffix for the Github OAuth Token for Cloudformation Stack {{cookiecutter.project_name.lower().replace(' ', '-')}}-pipeline" \
```

Comment:
Maybe it's just me but I find that some explanation as to why we need the Suffix would help understanding why this step and the CloudFormation param are required.